app/src/components/inputs/inputDropdown/inputDropdown.jsx

```diff
@@ -75,6 +77,13 @@ export class InputDropdown extends Component {
     this.node = node;
   };
 
+  getOptionChangeHandler = (option) =>
+    (!option.disabled &&
```

Comment:
I think that using `if..else` will be more verbose, but more readable here.

Reply:
Fixed